extensions/camel-k/client/src/main/java/io/fabric8/camelk/client/util/ApiVersionUtil.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.camelk.client.util;
+
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.fabric8.kubernetes.client.utils.Utils;
+
+/** Borrowed from the client. */
+public class ApiVersionUtil {
```

Comment:
This should probably be moved outside of the client project indeed at some point… :(

Reply:
Maybe move this to the kubernetes-model-common module?